chainstate/test-suite/src/tests/double_spend_tests.rs

```diff
@@ -224,9 +224,8 @@ fn overspend_single_output(#[case] seed: Seed) {
                 .build_and_process()
                 .unwrap_err(),
             ChainstateError::ProcessBlockError(BlockError::StateUpdateFailed(
-                ConnectTransactionError::AttemptToPrintMoney(
-                    Amount::from_atoms(tx1_output_value),
-                    Amount::from_atoms(tx2_output_value)
+                ConnectTransactionError::IOPolicyError(
+                    chainstate::IOPolicyError::AttemptToPrintMoneyOrViolateTimelockConstraints
```

Comment:
This loss of information is very unfortuante... we should try to do something about it.

Reply:
change the order of call so that we got prev error first